completion/git-machete.completion.bash

```diff
@@ -14,7 +14,7 @@ _git_machete() {
   local shells="bash fish zsh"
 
   local common_opts="--debug -h --help -v --verbose"
-  local add_opts="-o --onto= -R --as-root -y --yes"
+  local add_opts="-o --onto= -R --as-root -y --yes -f --first"
```

Comment:
Pls reorder to keep abc order, here and in other completion files

Reply:
Done -- here and the other files!